<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditHotkeysDialog</class>
 <widget class="QDialog" name="EditHotkeysDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Hotkeys</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>20</y>
     <width>361</width>
     <height>372</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <property name="spacing">
     <number>20</number>
    </property>
    <item row="1" column="2">
     <widget class="QToolButton" name="toolButton_2">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/Resources/ic_reply_black_24px.svg</normaloff>:/icons/Resources/ic_reply_black_24px.svg</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QToolButton" name="toolButton_5">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/Resources/ic_reply_black_24px.svg</normaloff>:/icons/Resources/ic_reply_black_24px.svg</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QToolButton" name="toolButton_6">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/Resources/ic_reply_black_24px.svg</normaloff>:/icons/Resources/ic_reply_black_24px.svg</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QKeySequenceEdit" name="keySequenceEditSwitchToNextServer"/>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="toolButton_3">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/Resources/ic_reply_black_24px.svg</normaloff>:/icons/Resources/ic_reply_black_24px.svg</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Switch system proxy mode</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/Resources/ic_reply_black_24px.svg</normaloff>:/icons/Resources/ic_reply_black_24px.svg</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Show Logs</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QKeySequenceEdit" name="keySequenceEditShowLogs"/>
    </item>
    <item row="3" column="2">
     <widget class="QToolButton" name="toolButton_4">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../../icons.qrc">
        <normaloff>:/icons/Resources/ic_reply_black_24px.svg</normaloff>:/icons/Resources/ic_reply_black_24px.svg</iconset>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Switch to prev server</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Switch to next server</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QKeySequenceEdit" name="keySequenceEditSwitchShareOverLAN"/>
    </item>
    <item row="4" column="1">
     <widget class="QKeySequenceEdit" name="keySequenceEditSwitchToPrevServer"/>
    </item>
    <item row="0" column="1">
     <widget class="QKeySequenceEdit" name="keySequenceEditSwitchSystemProxy"/>
    </item>
    <item row="1" column="1">
     <widget class="QKeySequenceEdit" name="keySequenceEditSwitchSystemProxyMode"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Switch share over LAN</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Switch system proxy</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRegAll">
        <property name="text">
         <string>Reg All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonConfirm">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>keySequenceEditSwitchSystemProxy</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>keySequenceEditSwitchSystemProxyMode</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_3</sender>
   <signal>clicked()</signal>
   <receiver>keySequenceEditSwitchShareOverLAN</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_4</sender>
   <signal>clicked()</signal>
   <receiver>keySequenceEditShowLogs</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_5</sender>
   <signal>clicked()</signal>
   <receiver>keySequenceEditSwitchToPrevServer</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_6</sender>
   <signal>clicked()</signal>
   <receiver>keySequenceEditSwitchToNextServer</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
